### ISSUE TITLE
compilation of zmq 4.2.0 on arch linux fails.

### ISSUE THREAD

**Issue by bl0x** [FairRootGroup/FairSoft#57]
Solution is to use the newest version 4.2.2.
The problem lies within zmq itself and has to do with improper autoconf.
Please refer to:
https://github.com/zeromq/libzmq/pull/2206

**Comment by ironMann** [comment]
The same with Centos7 and gcc6.3. I'll post my simple workarrund here, since I'm not familiar with this build flow to implement a proper fix: 

```diff
diff --git scripts/install_zeromq.sh scripts/install_zeromq.sh
index b344799..31283a9 100644
--- scripts/install_zeromq.sh
+++ scripts/install_zeromq.sh
@@ -37,7 +37,7 @@ then
 #      PKG_CONFIG_PATH=$SIMPATH_INSTALL/lib/pkgconfig ./configure --prefix=$install_prefix --libdir=$install_prefix/lib --enable-static
 #    fi

-    make
+    make LDFLAGS=-ldl
     make install

     check_all_libraries  $install_prefix/lib
```

**Comment by MohammadAlTurany** [comment]
solved in PR #63
